.travis.yml

```diff
@@ -9,6 +9,7 @@ rvm:
   - 2.1.10
   - 2.2.5
```

Comment:
@matthewrudy might as well bump others to latest patches, 2.2.7 and 2.3.4.  They are available as binary downloads on travis via RVM so won't affect the build speed.